Implement first version of graphql subscriptions mock with Playwright adapter (#1)
foundation-zero/graphql-subscriptions-mock: README.md

### GraphQL subscriptions mock

A GraphQL subscriptions mocking library, including an adapter for Playwright.

> An exhaustive README will follow with the publishing of this package.

Reviewer: ```suggestion
A comprehensive README will follow with the publishing of this package.
```
Manage expectations
